packages/components/socialFeed/SocialCard/cards/SocialArticleMarkdownCard.tsx

```diff
@@ -0,0 +1,197 @@
+import { LinearGradient } from "expo-linear-gradient";
+import React, { FC, memo, useEffect, useState } from "react";
+import { StyleProp, useWindowDimensions, View, ViewStyle } from "react-native";
+
+import { BrandText } from "../../../BrandText";
+import { OptimizedImage } from "../../../OptimizedImage";
+import { CustomPressable } from "../../../buttons/CustomPressable";
+import { SpacerColumn } from "../../../spacer";
+import { FlaggedCardFooter } from "../FlaggedCardFooter";
+import { SocialCardFooter } from "../SocialCardFooter";
+import { SocialCardHeader } from "../SocialCardHeader";
+import { SocialCardWrapper } from "../SocialCardWrapper";
+
+import { Post } from "@/api/feed/v1/feed";
+import defaultThumbnailImage from "@/assets/default-images/default-article-thumbnail.png";
+import { useAppNavigation } from "@/hooks/navigation/useAppNavigation";
+import { zodTryParseJSON } from "@/utils/sanitize";
+import {
+  ARTICLE_THUMBNAIL_IMAGE_MAX_WIDTH,
+  SOCIAl_CARD_BORDER_RADIUS,
+} from "@/utils/social-feed";
+import {
+  neutral00,
+  neutral33,
+  neutralA3,
+  withAlpha,
+} from "@/utils/style/colors";
+import { fontRegular13, fontRegular15 } from "@/utils/style/fonts";
+import {
+  layout,
+  RESPONSIVE_BREAKPOINT_S,
+  SOCIAL_FEED_BREAKPOINT_M,
+} from "@/utils/style/layout";
+import { ZodSocialFeedArticleMetadata } from "@/utils/types/feed";
+
+const ARTICLE_CARD_PADDING_VERTICAL = layout.spacing_x2;
+const ARTICLE_CARD_PADDING_HORIZONTAL = layout.spacing_x2_5;
+
+export const SocialArticleMarkdownCard: FC<{
+  post: Post;
+  isPostConsultation?: boolean;
+  style?: StyleProp<ViewStyle>;
+  refetchFeed?: () => Promise<any>;
+  isFlagged?: boolean;
+  disabled?: boolean;
+}> = memo(
+  ({ post, isPostConsultation, refetchFeed, style, isFlagged, disabled }) => {
+    const navigation = useAppNavigation();
+    const [localPost, setLocalPost] = useState<Post>(post);
+    const [viewWidth, setViewWidth] = useState(0);
+    const { width: windowWidth } = useWindowDimensions();
+
+    const articleCardHeight =
+      windowWidth < SOCIAL_FEED_BREAKPOINT_M ? 214 : 254;
+    const thumbnailImageWidth = viewWidth / 3;
+    const borderRadius =
+      windowWidth < RESPONSIVE_BREAKPOINT_S ? 0 : SOCIAl_CARD_BORDER_RADIUS;
+
+    const metadata = zodTryParseJSON(
+      ZodSocialFeedArticleMetadata,
+      localPost.metadata,
+    );
+    const thumbnailImage = metadata?.thumbnailImage;
+    const shortDescription = metadata?.shortDescription || "";
+    const title = metadata?.title;
+
+    useEffect(() => {
+      setLocalPost(post);
+    }, [post]);
+
+    const thumbnailURI = thumbnailImage?.url
+      ? thumbnailImage.url.includes("://")
+        ? thumbnailImage.url
+        : "ipfs://" + thumbnailImage.url // we need this hack because ipfs "urls" in feed are raw CIDs
+      : defaultThumbnailImage;
```

Comment:
can we remove this hack now?

Reply:
Yes, we have `OptimizedImage`
https://github.com/TERITORI/teritori-dapp/pull/1451/commits/9c42f452b7854b855dffac21b1c61a3212b74a0e#diff-2322effa5ea2906ae44dbd3959253b8195a179f73e4233c92f49eeb781a7fe19R71